### ISSUE TITLE
Configure nvenc for better quality picture

### ISSUE THREAD

**Issue by 0xd34d10cc** [0xd34d10cc/shar#269]
By default it displays a pixelated mess

**Comment by 0xd34d10cc** [comment]
Could be a duplicate of #139

**Comment by 0xd34d10cc** [comment]
Doesn't seem so. After #281 the quality is better, but it is still much worse then x264.
